ouroboros-network/ouroboros-network.cabal

```diff
@@ -222,11 +222,8 @@ library
 library protocol-tests
   hs-source-dirs:      protocol-tests
   default-language:    Haskell2010
-  exposed-modules:     Ouroboros.Network.Protocol.BlockFetch.Direct
-                       Ouroboros.Network.Protocol.BlockFetch.Examples
+  exposed-modules:     Ouroboros.Network.Protocol.BlockFetch.Examples
```

Comment:
can this commit ( commit c9eb7f1918fc9a6b27774e949272b3f83dd541c1 ) be squashed with all other `ouroboros-protocol-tests: updated against typed-protocols changes` commits?

Reply:
There's only one commit which updates `ouroboros-protocol-tests`.

Reply:
Oh I think I know what you mean, yes it makes sense to squash the two commits.